src/mono/mono/mini/interp/transform.c

```diff
@@ -3576,6 +3576,12 @@ interp_field_from_token (MonoMethod *method, guint32 token, MonoClass **klass, M
 	return field;
 }
 
+static void 
+add_bb_to_basic_blocks(TransformData *td, InterpBasicBlock *bb)
```

Comment:
I don't really see the usefulness of this method

Reply:
I removed the function.****